fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/ParameterVisitorBatchDAO.java

```diff
@@ -370,9 +394,11 @@ public void visit(TokenParmVal param) throws FHIRPersistenceException {
         String tokenValue = param.getValueCode();
         try {
             int parameterNameId = getParameterNameId(parameterName);
-            int codeSystemId = getCodeSystemId(codeSystem);
 
+            // TODO handle base (non-resource-specific) token values for issue #1366
```

Comment:
maybe I'm missing this one? What does this todo mean?

Reply:
TODO was left over. Removed.